### PR TITLE
[ado] Pierce ADO NPM feed cache for published packages

### PR DESCRIPTION
**Pull request by alloy**
Still need to add the PAT to the secrets group, looking into ADO CLI now.

### CODE REVIEW

File: .azure-devops/graphitation-release.yml [hunk 1]
```diff
@@ -41,3 +43,7 @@ jobs:
         displayName: Configure git for release
       - script: yarn release -y -n $(ossNpmToken) --access public
         displayName: Release
+      - template: ./steps/pierce-ado-npm-mirror-cache.yml
+        parameters:
+          adoNpmFeedPat: $(adoNpmFeedPat)
```
Comment (alloy) [inline .azure-devops/graphitation-release.yml:48]: Is this the correct way to make a secret available to a template?

**Review by vejrj — APPROVED** [review]

**Comment by alloy** [comment]
@kenotron I'd also like to make sure this step does not in any way fail the pipeline, as it needs some testing and isn't a required step. Do you know how I can ensure that?

File: .azure-devops/graphitation-release.yml (hunk 1 above)
Comment (kenotron) [reply .azure-devops/graphitation-release.yml:48]: try just using "ossNpmToken" here.

Comment (alloy) [reply .azure-devops/graphitation-release.yml:48]: @kenotron So from our offline chat I assume you now understand that this isn't what we're after.

Comment (kenotron) [reply .azure-devops/graphitation-release.yml:48]: I'll add this for you :)

**Comment by alloy** [comment]
At least in a test with no publishing it doesn’t fail and if it were to fail after the next publish, it would at least fail _after_ the publish.

**Pull request merged**